carbon-design-system/ibm-security
# fix(panel): add a11y props and attributes

## Affected issues

- Resolves #278 

(NOTE: I went ahead and made changes off of `dev` instead of just off `2.x)

## Proposed changes

- use same a11y props and attributes that Carbon uses in the `Modal` component: https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Modal/Modal.js 

## Testing instructions

- Run the storybook deployment in Chrome, view `PanelV2` and `Panel` in isolation, and then run the DAP tool to ensure that there are no violations (I suggested running when closed & then rerunning when open)
